Tests/mocks/base64.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
for RDKShell , please replace trower-base64 with WPEFramework built-in alternative - Core::ToString. this will allow to remove trower-base64 from RDKShell/CMakeLists.txt, and remove this file and its mock

Reply:
Ezhil experimented using WPEFramework ToString, but for a 1080p image from RDKShell compositor, the image data size needs a uint32. WPEFramework ToString supports takes a uint16, so we cannot use this. 
` void ToString(const uint8_t object[], const uint16_t length, const bool padding, string& result) `
Can we take removing trower dependency later.

Reply:
I think this is a matter of refactoring. As a first step it can be a wrapper with the same interface and unit tests for it. This should allow to remove trower-base64 from builds, reduce image size, simplify testing